.circleci/config.yml

```diff
@@ -0,0 +1,32 @@
+version: 2
+
+jobs:
+  build:
+    docker:
+      - image: circleci/python:3.7.3-stretch
+      - image: wurstmeister/zookeeper
+        name: zookeeper
+      - image: wurstmeister/kafka:2.12-2.1.0
+        name: kafka
+        environment:
+          KAFKA_ADVERTISED_HOST_NAME: kafka
```

Comment:
How would we test any extra brokers? Just add the env vars?

Reply:
As discussed, we would have to add more images to circleci so that our brokers are agnostic to the client which communicates with them.